documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -339,49 +329,92 @@ Model In Image model files, application archives, and the WDT installation files
    In a later step, we will specify a domain resource `domain.spec.configuration.model.modelHome`
    attribute that references this directory.
 
-1. Run `docker build` using `/tmp/mii-sample/ai-docker-file/Dockerfile` to create your auxiliary
-   image using a small `busybox` image as the base image.
+1. Use one of the two options listed below to create the auxiliary image using a small `busybox` image as the base image. 
+   - Option 1 - Use the `createAuxImage` option of the [_WebLogic Image Tool_ (WIT)](https://oracle.github.io/weblogic-image-tool/userguide/tools/create-aux-image/) to create the auxiliary image. Run the following command:
+
+     ```shell
+     $ /tmp/mii-sample/model-images/imagetool/bin/imagetool.sh createAuxImage \
+       --tag model-in-image:WLS-AI-v1 \
+       --wdtModel ./models/model.10.yaml \
+       --wdtVariables ./models/model.10.properties \
+       --wdtArchive ./models/archive.zip
+     ```
+  
+     If you don't see the `imagetool` directory under `/tmp/mii-sample/model-images` or the WDT installer is not in the ImageTool cache, then you have skipped step 9 in the [prerequisites]({{< relref "/samples/domains/model-in-image/prerequisites.md" >}}).
+ 
+     In a later step, we will specify corresponding domain resource attributes:
+     - `domain.spec.configuration.model.wdtInstallHome`
+       so that it references the WDT installation directory,
+       which the `imagetool.sh createAuxImage` command
+       defaults to `/auxiliary/weblogic-deploy`
+       (where `/auxiliary` is the default `--wdtHome` value for the `imagetool.sh createAuxImage` command).
+     - `domain.spec.configuration.model.modelHome`
+       so that it matches the `imagetool.sh createAuxImage` command's `--wdtModelHome` value,
+       which defaults to `/auxiliary/models`.
+
+   - Option 2 - Alternatively, you can create the auxiliary image manually by following these steps.
+
+     - First, install WDT in the staging directory and remove its `weblogic-deploy/bin/*.cmd` files, which are not used in UNIX environments:
+       ```shell
+       $ unzip /tmp/mii-sample/model-images/weblogic-deploy.zip -d .
+       $ rm ./weblogic-deploy/bin/*.cmd
+       ```
+       In a later step, we will specify a domain resource `domain.spec.configuration.model.wdtInstallHome`
+         attribute that references this WDT installation directory.
+    
+       If the `weblogic-deploy.zip` file is missing, then you have skipped a step in the prerequisites.
```

Comment:
which step was skipped?  Be explicit, if possible. Or say something like, "If .... missing, please repeat step X."

Reply:
If the steps are ever renumbered, then this will become out of date. Better to (very briefly) describe the step than to give an actual number...

Reply:
Changed.